tsconfig.typedoc.json

```diff
@@ -1,5 +1,11 @@
 {
-  "exclude": ["website", "integration-tests", "apps"],
+  "exclude": [
+    "website",
+    "integration-tests",
+    "apps",
+    "**/test/**",
+    "**/*.spec.ts"
```

Comment:
thought there is a separate pr for this change already
Would be great to add "example" since we only use typedoc comments on functions exported under packages/**/src/** to compile typedoc page